### ISSUE TITLE
UNPACK false positive, unused variables

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1860]
```abap
DATA foo TYPE i.
DATA bar TYPE c LENGTH 10.
UNPACK foo TO bar.
```

![image](https://user-images.githubusercontent.com/5888506/111155730-1f7f7480-8595-11eb-86d8-25242d701a1b.png)
